src/main/java/entities/Post.java

```diff
@@ -80,11 +80,10 @@ public LocalDate getDeadline(){
     }
 
```

Comment:
Could we also create a getter method for title?

Reply:
I added one in my PR for filter post but this will probably get pushed before so its best to add that change here so it gets added earlier

Reply:
ok i will check it and add it